packages/generator/src/renders/text.ts

```diff
@@ -31,8 +31,8 @@ import {
   convertSchemaDimensionsToPt
 } from '../renderUtils'
 
-const getFontProp = async ({ input, font, schema }: { input: string, font: Font, schema: TextSchema }) => {
-  const fontSize = schema.dynamicFontSize ? await calculateDynamicFontSize({ textSchema: schema, font, input }) : schema.fontSize ?? DEFAULT_FONT_SIZE;
+const getFontProp = async ({ value, font, schema }: { value: string, font: Font, schema: TextSchema }) => {
+  const fontSize = schema.dynamicFontSize ? await calculateDynamicFontSize({ textSchema: schema, font, input: value }) : schema.fontSize ?? DEFAULT_FONT_SIZE;
```

Comment:
we could rename the arguments on `calculateDynamicFontSize` in future to align with this renaming 👍